src/data-mgt/python/events-monitor/main.py

```diff
@@ -0,0 +1,100 @@
+import json
+import os
+from datetime import datetime, timedelta
+
+import requests
+
+DEVICE_REGISTRY_BASE_URL = os.getenv("DEVICE_REGISTRY_URL", "http://staging-platform.airqo.net/api/v1/")
+TENANT = os.getenv("TENANT", "airqo")
+SLACK_WEBHOOK = os.getenv("SLACK_WEBHOOK", "")
```

Comment:
Do we need to have this ENVs somewhere or are we supposed to enter then in the CLI? @NoahNsimbe

Reply:
@Baalmart you are supposed to enter them in the CLI, currently, they are only defined in the deployment file